particles/PipeApps/AndroidAutofill.arcs

```diff
@@ -5,7 +5,7 @@ schema AutofillRequest
   Text hint
 
 // An Android particle which powers the Android AutofillService.
-particle AutofillParticle
+particle AutofillParticle in 'java'
```

Comment:
This will cause the manifest tests to fail.. Probably better to have them fail so that the demo works again. I'll work on fixing them up.

Reply:
i ran tests locally and they all passed... i think..

Reply:
`bazel test particles/PipeApps/...` will fail I think